README.md

```diff
@@ -97,156 +116,101 @@ For example:
 }
 ```
 
-#### config-files/{env}/tables.json
-All table's primary key, columns, index information will be stored here.
-It will be required for query and update both type of operation to get primary key, sort or any other index present.
-It helps to query data on primary key or sort key.
-It helps to update data based on primary key or sort key.
-It will be similar to dynamodb table's architecture.
+### config-files/{env}/tables.json
 
-| Key | Used For |
-| ------ | ------ |
-| tableName | table name present in dynamoDb |
-| partitionKey | Primary key |
-| sortKey| Sorting key |
-| attributeTypes | Column names and type present |
-| indices | indexes present in the table |
+`tables.json` contains the description of the tables as they appear in DynamoDB. This includes all table's primary key, columns and index information. This file supports the update and query operations by providing the primary key, sort key and any other indexes present.
 
+| Key               | Description
+| ----------------- | -----------
+| tableName         | Name of the table in DynamoDB
+| partitionKey      | Primary key of the table in DynamoDB
+| sortKey           | Sorting key of the table in DynamoDB
+| attributeTypes    | Key/Value list of column names and type
+| indices           | Collection of index objects that represent the indexes present in the DyanmoDB table
 
 For example:
 
-```
+```json
 {
     "tableName":{
         "partitionKey":"primary key or Partition key",
```

Comment:
Not a breaking change but while you are at fixing other comments, can we add a space after the colon to make it consistent with other json data?